spec/forms/admin/adjustments/add_another_adjustment_form_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Admin::Adjustments::AddAnotherAdjustmentForm, type: :model do
+  include Rails.application.routes.url_helpers
```

Comment:
Less objection here but still not a fan, how about

```ruby
let(:urls) { Rails.application.routes.url_helpers }

it { is_expected.to eq urls.some_statement_path }
```

Reply:
actually, moving the URLs out into the controller means I don't need this include any more